client/templates/juror-management/juror-record/juror-history.njk

```diff
@@ -0,0 +1,53 @@
+{% extends "layouts/default.njk" %}
+
+{% from "moj/components/sub-navigation/macro.njk" import mojSubNavigation %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/tag/macro.njk" import govukTag %}
+
+{% block page_title %}{{ serviceName }} - Juror record - History{% endblock %}
+{% block page_identifier %}Juror record - History{% endblock %}
+
+{% set currentApp = "Juror management" %}
+
+{% block beforeContent %}
+  {% include "includes/back-link.njk" %}
+{% endblock %}
+
+{% block content %}
+  {% set currentName = [juror.commonDetails.title, juror.commonDetails.firstName, juror.commonDetails.lastName] %}
+
+  {% include "../_partials/heading.njk" %}
+
+  <h2>
+    History
+  </h2>
```

Comment:
probably needs a `govuk-caption-(m|l)` class

Reply:
Fixed - used the govuk-heading-l to match the other juror record tabs